src/main/resources/specification/terms-and-conditions.yaml

```diff
@@ -0,0 +1,138 @@
+  url: https://informasjonsforvaltning.github.io
+tags:
+  - name: organization-terms
+    description: Organization terms and conditions
+paths:
+  /terms/status/org/{id}:
+    get:
+      tags:
+        - organization-terms
+      summary: Get terms and conditions acceptation status for an organization
+      description: Returns organization terms and conditions status
+      operationId: statusOrgTerms
+      parameters:
+        - name: id
+          in: path
+          description: organization id
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            text/plain:
+              schema:
+                type: boolean
+  /terms/org/{id}:
+    get:
+      security:
+        - bearerAuth: []
+      tags:
+        - organization-terms
+      summary: Get organization terms and conditions info.
+      description: Returns organization terms and conditions info
+      operationId: getOrgTerms
+      parameters:
+        - name: id
+          in: path
+          description: organization id
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/OrgAcceptions"
+        '404':
+          description: Organization has not accepted the terms and conditions
+    put:
+      security:
+        - bearerAuth: []
+      tags:
+        - organization-terms
+      summary: Accept organization terms and conditions
+      description: Accept terms and conditions for the organization
+      operationId: acceptOrgTerms
+      parameters:
+        - name: id
+          in: path
+          description: organization id
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/OrgAcceptions"
+    delete:
+      security:
+        - bearerAuth: []
+      tags:
+        - organization-terms
+      summary: Remove organization acceptation history
+      description: Delete accept history for the organization
+      operationId: deleteOrgTerms
+      parameters:
+        - name: id
+          in: path
+          description: organization id
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+components:
+  schemas:
+    OrgAcceptations:
+      type: object
+      description: Information regarding the acceptation of an organizations terms and conditions
+      properties:
+        orgnr:
+          type: string
+          description: Organization id
+        acceptations:
+          type: array
+          description: List of acceptations of terms and conditions
```

Comment:
Kvifor skal vi ha ei liste her av terms and conditions? Er det 1:m på denne relasjonen?

Reply:
Om vi f.eks. oppdaterer vilkårene og de må godkjenne på ny, regner med at vi ikke bare skal slette hvem som godtok første gangen?

Reply:
Har avklart med Joachim og Lars: vi skal ikkje lagre forrige godkjenning. Så vi trenger ikkje liste. Men vi må innføre eit felt som seier kva for versjon (semantisk versjonering) av terms som er godtatt.